### ISSUE TITLE
Rec 识别模型训练， gpu 利用率不高

### ISSUE THREAD

**Issue by frkhit** [PaddlePaddle/PaddleOCR#3679]
训练 PaddleOCR 识别模型  ，GPU 利用率不高 ，而 cpu 4 个 核心 利用率 均在 50%左右。

环境:
单 GPU， 显存 16GB， 4 核心 cpu。
num_workers = 4。
batch_size_per_card=256 （已经是最大了）。 

问题：
1） 有什么办法提高 gpu 利用率？ 这个教程看了， https://www.paddlepaddle.org.cn/documentation/docs/zh/1.5/advanced_usage/best_practice/dist_training_gpu.html， 还是没头绪。

2） 现在怀疑， gpu 利用率不高， 是因为 cpu 加载数据集的时候， 执行了 transforms 的几个 op 。请问这些 op ， 能使用 gpu 执行吗？

**Comment by frkhit** [comment]
训练参数部分配置如下：


Train:
  dataset:
    name: SimpleDataSet
    data_dir: /dataset/
    label_file_list: ["/dataset/lsvt/rec_gt_train.txt",]
    ratio_list: [1.0,]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - RecAug:
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 320]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 128
    drop_last: True
    num_workers: 4

**Comment by LDOUBLEV** [comment]
> 2） 现在怀疑， gpu 利用率不高， 是因为 cpu 加载数据集的时候， 执行了 transforms 的几个 op 。请问这些 op ， 能使用 gpu 执行吗？

这个部分目前只能在CPU上执行
如果你的场景比较简单，可以去掉一些数据增强；

**Comment by frkhit** [comment]
@LDOUBLEV 好的， 我试试
